projects/unix/tomviz.bundle.cmake

```diff
@@ -32,34 +38,75 @@ install(CODE
       -P ${CMAKE_CURRENT_LIST_DIR}/install_dependencies.cmake)"
   COMPONENT superbuild)
 
-if (qt_ENABLED AND NOT USE_SYSTEM_qt)
-  install(DIRECTORY
-    # install all qt plugins (including sqllite).
-    # FIXME: we can reconfigure Qt to be built with inbuilt sqllite support to
-    # avoid the need for plugins.
-    "${install_location}/plugins/"
-    DESTINATION "lib/tomviz"
-    COMPONENT superbuild
-    PATTERN "*.a" EXCLUDE
-    PATTERN "tomviz-${tomviz_version}" EXCLUDE
-    PATTERN "fontconfig" EXCLUDE
-    PATTERN "*.jar" EXCLUDE
-    PATTERN "*.debug.*" EXCLUDE
-    PATTERN "libboost*" EXCLUDE)
-  install(DIRECTORY "${install_location}/share/appdata"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
-  install(DIRECTORY "${install_location}/share/applications"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
-  install(DIRECTORY "${install_location}/share/icons"
-    DESTINATION "share"
-    USE_SOURCE_PERMISSIONS
-    COMPONENT superbuild)
+if (qt_ENABLED)
+  set(qt_plugins)
+  if (NOT USE_SYSTEM_qt)
+    file(GLOB qt_plugins ${install_location}/plugins/*)
+  else()
+    file(GLOB qt_plugins ${Qt5_DIR}/../../../plugins/*)
+  endif()
+  foreach(plugin ${qt_plugins})
+    if(IS_DIRECTORY ${plugin})
+      install(DIRECTORY ${plugin}
+        DESTINATION "plugins"
+        COMPONENT superbuild
+        PATTERN "*.a" EXCLUDE
+        PATTERN "tomviz-${tomviz_version}" EXCLUDE
+        PATTERN "fontconfig" EXCLUDE
+        PATTERN "*.jar" EXCLUDE
+        PATTERN "*.debug.*" EXCLUDE
+        PATTERN "libboost*" EXCLUDE)
+    endif()
+  endforeach()
+  if (NOT USE_SYSTEM_qt)
```

Comment:
> does this mean the code only works for non-system Qt at this stage?

Since the binary I sent use was built against a system Qt, I think it should work for both.  The old code only worked for non-system Qt and I made the modifications to install the plugins.  I wasn't sure what the rest of this was doing so I left it in the `if (NOT USE_SYSTEM_qt)` block.  It doesn't seem to be necessary, but I haven't tested a locally built Qt.  I don't think this had been updated since we switched to Qt5, so it may be Qt4-related code as well.